src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -187,8 +187,8 @@ GtProcessDock::GtProcessDock() :
             SLOT(pasteElement(QModelIndex)));
     connect(m_view, SIGNAL(runTaskElement(QModelIndex)),
             SLOT(runProcess()));
-    connect(m_view, SIGNAL(skipCalcultorElement(QModelIndex,bool)),
-            SLOT(skipComponent(QModelIndex,bool)));
+    connect(m_view, SIGNAL(skipCalcultorElement(const QList<QModelIndex>&,bool)),
+            SLOT(skipComponents(const QList<QModelIndex>&,bool)));
```

Comment:
You may want to use this notation instead, that way it does not matter which arguments the functions have or if they are changing
```suggestion
    connect(m_view, &GtProcessView::skipCalcultorElement,
                 this, &GtProcessDock::skipComponents));
```

Reply:
@mariusalexander I think this is only possible for static functions, isn't it?